### PR TITLE
Feature/upgrade to sb6

### LINKED ISSUE #1774: [Child of #1773] Update Storybook to version 6 [Due Date: 03.08.20]
We dont want to redesign storybook on a old version to avoid doubled work. The updated causes some breaking changes and we want to fix them first.

### LINKED ISSUE #1877: Testimonials: Knobs not working correctly
Maybe first solve #1797 ?!

Maybe just at Testimonials - not sure. No error message.

Problem:
Open Testimonials: https://patterns.axa.ch/?path=/story/components-testimonials--story
Click "showallinline" -> works
Click again -> nothing happens
Click to another component -> Knobs window of Testimonials stays

Solution:
- Update to the new plugin as mentioned here: https://github.com/storybookjs/storybook/issues/8614#issuecomment-635502840

- Update Storybook #1774 . Because this bug is gone at https://axa-ch.github.io/plib-feature/feature/update-storybook-5-3/?path=/story/components-testimonials--story

### LINKED ISSUE #1900: Storybook: hot-reload shows SB error
Since reordering the Stories (Raphi S. PR) the hot reload is not working any more. Because the structure of SB stories changed.

- Maybe it is fixed with SB 6. #1774 
- Maybe use the new story format fixes that https://storybook.js.org/docs/formats/storiesof-api/#component-story-format-migration
- maybe revert raphis change d118f2d9

### PR DESCRIPTION
**Pull request by danyball**
Fixes #1774
fixes #1877
fixes #1900

Info: Many files have changed because i switched the "old" story.js format to the new one "CSF". You do not have to review ALL the story/demo files. But please have a look at all other changes.

# unable to fix:
- 'storybook-**readme**/register': 3rd party addon. Not working correctly with SB6. Nobody maintains this Addon.
   - removed that completly

# wait to be fixed by others  (#1935):
- './addons/**codepreview**/register': custom addon but forked from a 3rd party addon. 
   - fixed that by reference directly to the 3rd party dependency
   - opened a issue to customise `HTML` Tab label: https://github.com/whitespace-se/storybook-addon-html/issues/27 
   - unfixed: this addon is not working with IE11. Maybe this problem is solved by checkin their `src` folder to our project. But than it is a "custom addon" and this causes another error (details below).
- './addons/**changelog**/register': custom addon. Seems to be a SB internal bug: https://github.com/storybookjs/storybook/issues/11255
- new Story format CSF always group by title. But we want to have a flat list. (Is also related to the alphabetical sorting.) --> https://github.com/storybookjs/storybook/issues/8255

# Done is when (DoD):
- [ ] Modifications within components are reflected by tests.
- [x] A self-review of the code has been performed.
- [ ] The modified component properties have been added to the typescript typings.
- [ ] Potential design changes have been reviewed by a designer.
- [x] The modifications are well documented (README.md, etc.) and showcased in the stories.
- [x] The modifications are documented in the code, particularly in hard-to-understand areas (focus on **WHY**).
- [ ] The modifications are shortly added to CHANGELOG.md with the issue number.
- [x] The modifications still work in IE.
- [x] The modifications are according to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
     "rollup-plugin-sass": "^1.1.0",
     "rollup-plugin-uglify": "^6.0.4",
     "sass-loader": "^7.1.0",
-    "storybook-readme": "5.0.6",
+    "storybook-readme": "5.0.8",
```
Comment (raphaellueckl) [inline package.json]: Das funktioniert nicht im IE. War zumindest bei mir immer so.

Comment (danyball) [reply package.json]: IE läuft. Aber alle nicht-offiziellen addons funktionieren nicht mehr.

**Comment by danyball** [comment]
- Remove readme code from stories
- fix codepreview code (disabling) at stories

**Comment by danyball** [comment]
TODO: make IE11 work great again:)

File: src/components/10-atoms/fieldset/ui.test.js [hunk 2]
```diff
@@ -67,7 +67,7 @@ test('should responsive stretch', async t => {
     ];
   });
 
-  const expected = JSON.stringify(['529px', '0px']); // 529px width of the radio correspond to 100% in a 575px window
+  const expected = JSON.stringify(['545px', '0px']); // 545px width of the radio correspond to 100% in a 575px window
```
Comment (raphaellueckl) [inline src/components/10-atoms/fieldset/ui.test.js:70]: The last number was forgotten, maybe just replace this comment with `100% width` or something.

**Review by raphaellueckl — APPROVED** [review]

File: src/components/10-atoms/fieldset/ui.test.js (hunk 2 above)
Comment (danyball) [reply src/components/10-atoms/fieldset/ui.test.js:70]: You mean 575px? Thats correct. At this test the browser is resized to 575px width.

**Pull request merged**